packages/fhir-converter/src/templates/cda/Sections/Procedures.hbs

```diff
@@ -11,6 +11,13 @@
                         {{>Resources/Organization.hbs org=procEntry.procedure.performer.assignedEntity.representedOrganization ID=(generateUUID (toJsonString procEntry.procedure.performer.assignedEntity.representedOrganization))}},
                         {{>References/Procedure/performer.actor.hbs ID=(generateUUID (toJsonString procEntry.procedure)) REF=(concat 'Organization/' (generateUUID (toJsonString procEntry.procedure.performer.assignedEntity.representedOrganization)))}},
                     {{/if}}
+                    
+                    {{#if procEntry.procedure.performer.assignedEntity}}
```

Comment:
Procedures missing provider details based on cx QA. Same format of pairing the inline reference to the practitioner with generating the practitioner. And of course, if the practitioner already exists we arent creating duplicates because of the handlebars merging